### ISSUE TITLE
Error on Yarn command: Request failed \"403 Forbidden\"". 

### ISSUE THREAD

**Issue by Safariba** [yarnpkg/yarn#5937]
**Do you want to request a *feature* or report a *bug*?**
Bug

**What is the current behavior?**
When running `Yarn`, an unexpected error occurs which causes the request to fail.

**If the current behavior is a bug, please provide the steps to reproduce.**

1- Open Command Prompt
2- Locate the folder which contains package.json file. .\Sources\WebApp\ClientApp
3- run command `yarn`
5- Encounter error on fetching packages:  

> An unexpected error occurred: "https://registry.yarnpkg.com/node-forge/-/node-forge-0.7.5.tgz: Request failed \"403 Forbidden\"". 

**What is the expected behavior?**
Install packages listed on package.json file 
**Please mention your node.js, yarn and operating system version.**

- node v8.11.2 (installed by installer (C:\Program Files\nodejs))
- yarn 1.7.0  (installed by installer (C:\Program Files (x86)\Yarn))
- 64-bit Operating System. Windows 10

Log:

> warning karma > log4js > nodemailer@2.7.2: All versions below 4.0.1 of Nodemailer are deprecated. See https://nodemailer.com/status/
> warning karma > log4js > loggly > request > node-uuid@1.4.8: Use uuid module instead
> warning karma > log4js > nodemailer > mailcomposer@4.0.1: This project is unmaintained
> warning karma > log4js > nodemailer > socks@1.1.9: If using 2.x branch, please upgrade to at least 2.1.6 to avoid a serious bug with socket data flow and an import issue introduced in 2.1.0
> warning karma > log4js > mailgun-js > proxy-agent > socks-proxy-agent > socks@1.1.10: If using 2.x branch, please upgrade to at least 2.1.6 to avoid a serious bug with socket data flow and an import issue introduced in 2.1.0
> warning karma > log4js > nodemailer > mailcomposer > buildmail@4.0.1: This project is unmaintained
> warning karma > socket.io > engine.io > uws@9.14.0: stop using this version
> error An unexpected error occurred: "https://registry.yarnpkg.com/node-forge/-/node-forge-0.7.5.tgz: Request failed \"403 Forbidden\"".

**Comment by rally25rs** [comment]
Are you still getting this error? It appears to be working for me right now. The npm registry has been a bit flakey since they moved to cloudflare last week, so this could have been a temporary problem.

**Comment by Safariba** [comment]
Yes. I still get this error and I realized that the `@angular/cli : ~1.7.0` is causing this error. If I update it to `6.0.1` for example, it succeed without problem. 

What is your global version of `angular/cli`?
